examples/atmajs/package.json

```diff
@@ -1,12 +1,12 @@
 {
   "private": true,
   "dependencies": {
-    "atma-class": "^1.0.68",
-    "includejs": "^0.9.10",
+    "atma-class": "^1.1.69",
+    "includejs": "^0.9.14",
     "jquery": "^2.1.3",
-    "maskjs": "^0.10.1",
-    "ruta": "^0.1.12",
-    "todomvc-common": "^1.0.1",
-    "todomvc-app-css": "^1.0.0"
+    "maskjs": "^0.51.26",
+    "ruta": "^0.1.16",
+    "todomvc-app-css": "^2.0.1",
+    "todomvc-common": "^1.0.2"
```

Comment:
For instance, assume my intent is to update the `todomvc-app-css` to the last minor version. In this case, I usually run `npm install todomvc-app-css@latest --save`: and it modifies the file. And am I understanding you right, your suggestion is: to set the versions in `package.json` manually to `{major}.{minor}.0` and afterwards just to run `npm install`?